Makefile

```diff
@@ -1,9 +1,19 @@
+TARGETS           = darwin/amd64 linux/amd64 windows/amd64
```

Comment:
I'm using [gox](https://github.com/mitchellh/gox) as a cross-build wrapper for go. The `build-cross` task doesn't use Docker, but a CI step definitely could. Do you mind giving an 👍 ?